tools/cli/go-whisk/whisk/client.go

```diff
@@ -47,6 +47,7 @@ const (
     DoNotProcessTimeOut = false
     ExitWithErrorOnTimeout = true
     ExitWithSuccessOnTimeout = false
+    UserAgent = "OpenWhisk-CLI/"
```

Comment:
@drcariel - after talking with @csantanapr, we think this would be better defined as a client.go externalized variable that could be overridden by client.go consumers.